storage/filesystem/module.go

```diff
@@ -0,0 +1,14 @@
+package filesystem
+
+import (
+	"srcd.works/go-git.v4/storage"
+	"srcd.works/go-git.v4/storage/filesystem/internal/dotgit"
+)
+
+type ModuleStorage struct {
+	dir *dotgit.DotGit
+}
+
+func (s *ModuleStorage) Module(name string) (storage.Storer, error) {
```

Comment:
Is already covered by the tests. And the documentation is on the interface as in the other methods